### PR TITLE
Properly handle ref-returning members when generating overrides.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/17698

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide

**Comment by tannergooding** [comment]
test windows_release_vs-integration_prtest please

**Comment by tannergooding** [comment]
test windows_debug_vs-integration_prtest please

File: src/VisualStudio/Core/Def/Implementation/PickMembers/PickMembersDialogViewModel.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ internal class PickMembersDialogViewModel : AbstractNotifyPropertyChanged
             IGlyphService glyphService,
             ImmutableArray<ISymbol> members)
         {
-            MemberContainers = members.Select(m => new MemberSymbolViewModel(m, glyphService)).OrderBy(s => s.MemberName).ToList();
+            MemberContainers = members.Select(m => new MemberSymbolViewModel(m, glyphService)).ToList();
```
Comment (DustinCampbell) [inline src/VisualStudio/Core/Def/Implementation/PickMembers/PickMembersDialogViewModel.cs]: This change doesn't look related. Was it intentional?

Comment (CyrusNajmabadi) [inline src/VisualStudio/Core/Def/Implementation/PickMembers/PickMembersDialogViewModel.cs]: It was.  I don't want the items sorted by name in the final UI.  I want them ordered as we found them in the inheritance hierarchy.

Comment (CyrusNajmabadi) [inline src/VisualStudio/Core/Def/Implementation/PickMembers/PickMembersDialogViewModel.cs]: That way you see all of 'objects' members grouped together, then the next highest derived type, etc. etc.

Comment (DustinCampbell) [inline src/VisualStudio/Core/Def/Implementation/PickMembers/PickMembersDialogViewModel.cs]: Oh, I agree. I was just pointing out how you were sneaking in design changes. Gotta keep my eye on you. :smile:

Comment (jasonmalinowski) [inline src/VisualStudio/Core/Def/Implementation/PickMembers/PickMembersDialogViewModel.cs]: @CyrusNajmabadi Can you add a comment stating the intent then? I could totally imagine somebody coming by, saying "heh, that's stupid it's not sorted", and adding in the exact thing you're removing.

(bonus points for having some sort of visible grouping so it's clear to users too.)
